File: Naloga 10/naloga_10.py
def naloga_10():
    vsota = 0
    while True:
        stevilo = int(input("vnesi stevilo: "))
        if stevilo  == 0:
            print(vsota)
            return
        elif stevilo > 33:
            vsota += stevilo
        else:
            continue
    return

if __name__ == '__main__':
    naloga_10()

File: Naloga 10/naloga_10.cpp
#include <iostream>

void naloga_10 () {
    int stevilo, vsota=0;
    do {
        std::cin >> stevilo;
        if (stevilo > 33) {
            vsota += stevilo;
        }
        else {
            continue;
        };
    } while(stevilo != 0);
    std::cout << "vsota: " << vsota << std::endl;
};

int main () {
    naloga_10();
    return 0;
};